sherpa/sim/sample.py

```diff
@@ -263,7 +269,7 @@ def get_scales(self,
             raise TypeError("scales option must be iterable of "
                             f"length {npar}")
 
-        return np.asarray(scales).transpose()
+        return np.asarray(scales)
```

Comment:
Why did you drop transpose() here?

Reply:
Because the transpose of a 1D NumPy array, which this one is, is the same array:

```
>>> xs = np.asarray([1, 2, 3])
>>> ys = np.asarray([1, 2, 3]).transpose()
>>> xs
array([1, 2, 3])
>>> ys
array([1, 2, 3])
>>> xs == ys
array([ True,  True,  True])
>>> xs.shape
(3,)
>>> ys.shape
(3,)
```